datarepo-clienttests/src/main/java/scripts/testscripts/DRSLookup.java

```diff
@@ -156,12 +164,15 @@ public void setup(List<TestUserSpecification> testUsers) throws Exception {
   public void userJourney(TestUserSpecification testUser) throws Exception {
     ApiClient apiClient = DataRepoUtils.getClientForTestUser(testUser, server);
     DataRepositoryServiceApi dataRepositoryServiceApi = new DataRepositoryServiceApi(apiClient);
-    DRSObject object = dataRepositoryServiceApi.getObject(dirObjectId, false);
-    logger.debug(
-        "Successfully retrieved drs object: {}, with id: {} and data project: {}",
-        object.getName(),
-        dirObjectId,
-        snapshotModel.getDataProject());
+    for (int i = 0; i < K; i++) {
```

Comment:
I'm curious: doesn't test runner have a way to look the user journey? If so, should we just use that?

Reply:
yes---but they dont always land at the same time. It's hard to line them all up for a maximum amplitude of hits, so by making a loop here, each user journey is longer and is more likely to overlap with all of the others